tokio/src/runtime/builder.rs

```diff
@@ -375,6 +385,30 @@ impl Builder {
             blocking_pool,
         })
     }
+
+    #[cfg(feature = "blocking")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "blocking")))]
+    /// Sets a custom timeout for a thread in the blocking pool.
+    /// By default, the timeout for a thread is constant and defined
+    /// in KEEP_ALIVE. In case the user wants to customize this
+    /// value, then they can use blocking_keep_alive() for it.
```

Comment:
nit:
```suggestion
    #[cfg_attr(docsrs, doc(cfg(feature = "blocking")))]
    /// Sets a custom timeout for a thread in the blocking pool.
    ///
    /// By default, the timeout for a thread is constant and defined
    /// in `KEEP_ALIVE`. In case the user wants to customize this
    /// value, then they can use `blocking_keep_alive()` for it.
```